app/web/features/dashboard/MyEvents.tsx

```diff
@@ -46,7 +42,7 @@ const useStyles = makeStyles((theme) => ({
   allUpcomingEventsLink: {
     justifySelf: "center",
   },
-  loaderContainer: {
+   loaderContainer: {
```

Comment:
It looks like we can get rid of this loaderContainer css class entirely now that you replaced it with `CenteredSpinner`?

Reply:
It is still being utilized as a container by the Button on line 104, but you're right it no longer is used a s a container for the spinner. Should I leave it but rename it so it's more clear?

Reply:
Ah yes please do rename it to something more relevant, thank you!